geospace-lib/src/main/scala/com/socrata/geospace/lib/regioncache/SpatialIndex.scala

```diff
@@ -113,7 +111,7 @@ object SpatialIndex {
   /**
    * Create a SpatialIndex[String] from a Layer/FeatureSource.  The feature ID will be stored in the index.
    *
-   * @param layer an [[org.geoscript.layer.Layer]] or GeoTools FeatureSource.
+   * @param layer an org.geoscript.layer.Layer or GeoTools FeatureSource.
```

Comment:
The original reason that I added the function findjar in BuildSettings. When I changed the [[[org.geoscript.layer.layer]] => org.geoscript.layer.layer originally others wanted me to undo and keep the link. So should we revert? I believe we can safely remove the additional (and cumbersome) code from BuildsSettings if we just not link.